todo_app/templates/index.html

```diff
@@ -11,8 +11,24 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for toDoItem in items %}
+          <li>{{ toDoItem.title}}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <form method="POST" action="/">
+        <div class="col-md-9 form-group row">
+          <label> New To-Do Item</label>
+          <input class="form-control" type="text" name="todoTitle">
```

Comment:
You could add a `required` attribute to prevent adding empty todos:
```suggestion
          <input class="form-control" type="text" name="todoTitle" required>
```